snapshot/git/gitdb/setup.go

```diff
@@ -21,6 +23,9 @@ func (m module) Install(b *ice.MagicBag) {
 		func(tmp *temp.TempDir) RepoIniter {
 			return &NewRepoIniter{tmp: tmp}
 		},
+		func() RepoUpdater {
+			return &NewRepoUpdater{}
```

Comment:
Can you add a github comment outlining the plan for the real impl? It'll be golang calling into a python script?